.fides/fides.toml

```diff
@@ -18,6 +18,8 @@ server_port = 8080
 
 [user]
 analytics_opt_out = false
+api_key = "test_api_key"
+encryption_key = "test_encryption_key"
```

Comment:
I'm trying to see where & how these are used... it looks like `user.api_key` isn't referenced anywhere, and `user.encryption_key` could safely be swapped out for `security.app_encryption_key`...

Reply:
@SteveDMurphy is that accurate? I believe both these configs are ones you added.

Reply:
Will look into `user.api_key` shortly, I think that one outdates me but as far as `user.encryption_key` I added that to encrypt user data at rest - I don't see any problem consolidating it to another key 👍🏽

Reply:
I see that `user.api_key` is wrapped up into the `request_headers` helper. So there are some codepaths there, just unclear on how that functions...

Reply:
`user.encryption_key` just needs to be changed out here -> https://github.com/ethyca/fides/blob/8b4e493fe77039ae580a5d7a90b2e257b17ed0ed/src/fides/api/ctl/sql_models.py#L67

The only usage I can see of `user.api_key` is in testing so probably safe to remove 👍🏽